### PR TITLE
Pod annotation configuration

### PR DESCRIPTION
**Pull request by lamht**
#### What this PR does / why we need it:
Add pod annotation for job and cronjob.
#### Which issue this PR fixes
https://github.com/helm/charts/issues/22472
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mysqldump]`)

### CODE REVIEW

**Review by monotek — APPROVED** [review]
/lgtm

**Review by monotek — APPROVED** [review]
/lgtm
/approve

File: stable/mysqldump/values.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ image:
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images#
   pullPolicy: IfNotPresent
 
+podAnnotations:
```
Comment (davidkarlsen) [inline stable/mysqldump/values.yaml:14]:
```suggestion
podAnnotations: {}
```

**Pull request closed without merging**